jupyterhub/bases/jupyterhub/configs/jupyterhub_config.py

```diff
@@ -1,11 +1,28 @@
 from kubernetes.client import V1Capabilities, V1SELinuxOptions
 spawner = c.OpenShiftSpawner
-def mcs_selinux_profile(spawner, pod):
+
+def idh_apply_pod_profile(spawner, pod):
+  """
+  This method is responsible for modifying the final notebook pod spec before submitting it to the k8s api
+  Modifying the pod based on the V1Pod and V1PodSpec avaialable in the kubernetes python API
+  https://github.com/kubernetes-client/python/blob/master/kubernetes/docs
+  """
+  # Pod container zero is the 'notebook' container with the env var we need to check
+  nb_container_env = pod.spec.containers[0].env
+
   # Apply profile from singleuser-profiles
   apply_pod_profile(spawner, pod)
```

Comment:
I forgot to move this to the first statement executed in the function as standard practice for calling the parent method AND the pod object will be modified by `apply_pod_profile` so we want to reference it after the parent modifies it